CHANGELOG.md

```diff
@@ -1,3 +1,32 @@
+# dbt_hubspot_source v0.19.0
+[PR #142](https://github.com/fivetran/dbt_hubspot_source/pull/142) includes the following updates:
+
+## Breaking Changes
+- Introduced new sources and their associated staging models to support expanded HubSpot data coverage:
+
+| Source Table                | Staging Model                                 | Enablement Variable                      |
```

Comment:
@fivetran-catfritz This looks very nice 😎 Thoughts on including umbrella variables that the staging models are also subject to? Like how `stg_hubspot__contact_form_submission` is disabled if `hubspot_marketing_enabled` set to False? Or `stg_hubspot__team_user` and `hubspot_team_enabled`?

Alternatively, we could just allude to umbrella variables and suggest looking at the README for more details on that, especially since some of these high-level variables aren't available in Quickstart